src/userstyles.yml

```diff
@@ -334,4 +340,11 @@ userstyles:
           answer: "It is there in-case you have an OLED display. If you have one, you might want to enable this."
       current-maintainers: [*isabelroses]
       past-maintainers: [*elkrien]
+  chess.com:
+    name: Chess.com
+    category: game
+    color: green
+    readme:
+      app-link: "https://chess.com"
+      current-maintainers: [ *skelebro1, *coldenate, *isabelroses ]
```

Comment:
Can you move this so its in alphabetical order please

Reply:
got it. I made the current maintainers in alphabetical order. 
you didn't mean all of the attributes right?